react/features/base/tracks/middleware.js

```diff
@@ -47,6 +53,17 @@ declare var APP: Object;
  */
 MiddlewareRegistry.register(store => next => action => {
     switch (action.type) {
+    case CONFERENCE_LEFT: {
```

Comment:
You are doing this here so the welcome page switches back to camera video, right? We need to handle that differently. First, all screen tracks should be destroyed when the conference ends and then IIRC the welcome page takes care of creating the video stream.